src/components/subscriptions/hooks/licenseManagerHooks.js

```diff
@@ -0,0 +1,168 @@
+import { useState, useEffect } from 'react';
+import qs from 'query-string';
+
+import LicenseManagerApiService from '../../../data/services/LicenseManagerApiService';
+import NewRelicService from '../../../data/services/NewRelicService';
+
+
+export const useSubscriptions = (includeInActive = false) => {
+  const [subscriptions, setSubscriptions] = useState({
+    results: [],
+    count: 0,
+    next: null,
+    previous: null,
+  });
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const [refreshCount, setRefreshCount] = useState(0);
+  const [pageUrl, setPageUrl] = useState(null);
+
+  const refreshSubscriptions = () => setRefreshCount(refreshCount + 1);
```

Comment:
I get the rationale for `refreshCount` (to force a re-render I'm guessing?) but it feels a bit odd to be necessary within a React hook.

Reply:
Yes, you are right. I have removed `refreshCount` and have also removed a lot of unnecessary code from the hooks. they look a lot cleaner now.